trunk/src/graphics/paraview/paraview_initialize.F90

```diff
@@ -98,6 +101,26 @@ subroutine paraview_begin(Id,Time)
       open(unit=PARAVIEW_IO,                                                  &
            file=trim(PARAVIEW_DIR)//trim(PREFIX)//"_"//trim(postfix)//'.vtu', &
            status = 'replace', form = 'unformatted', access = 'stream')
+      
+      open(unit = PVD_IO,file=trim(PARAVIEW_DIR)//trim(PREFIX)//'.pvd',status = 'replace', form = 'unformatted', access = 'stream')
+      write(PVD_IO) '<VTKFile type="Collection" version="1.0" byte_order="LittleEndian" header_type="UInt64">' //char(10)
```

Comment:
@ac1512 should this be `version="0.1"` ?

Reply:
When i take a bunch of adapted VTU meshes and ask paraview to save it into pvd files, it saves with version 1.0